src/api/operations/household.fragments.graphql

```diff
@@ -21,6 +21,9 @@ fragment HouseholdClientFields on HouseholdClient {
     externalIds {
       ...ClientIdentifierFields
     }
+    alerts {
+      ...ClientAlertFields
+    }
```

Comment:
Is this still needed? I wouldn't think so, now that you use a separate query to fetch the household alerts

Reply:
Hm, it seems this is indeed still needed bc in the enrollment modal, the `client` we have access to can be either a `ClientFieldsFragment` or `RecentHouseholdMember` (which is based on `HouseholdClientFieldsFragment`)